docs/reference/vectors/vector-functions.asciidoc

```diff
@@ -17,6 +16,8 @@ This is the list of available vector functions and vector access methods:
 6. <<vector-functions-accessing-vectors,`doc[<field>].vectorValue`>> – returns a vector's value as an array of floats
 7. <<vector-functions-accessing-vectors,`doc[<field>].magnitude`>> – returns a vector's magnitude
 
+NOTE: The `cosineSimilarity` and `dotProduct` functions are not supported for `bit` vectors.
```

Comment:
Thanks Ben!

Also  Line 196 says: The `hamming` function calculates {wikipedia}/Hamming_distance[Hamming distance] between a given query vector and document vectors. It is only **available for byte vectors.**.

We should correct this.